### ISSUE TITLE
PMLC 3.1.0 Crashes on Empty [options Node

### ISSUE THREAD

**Issue by tajmone** [pml-lang/pml-companion#98]
Converting the following PML document with empty `[options` node:

```pml
[doc [title Test] [options ]]
```

results in the following crash error:

```
ERROR:
The operation has been aborted due to the following unexpected application error:
java.lang.NullPointerException: Cannot invoke "dev.pp.parameters.textTokenParameter.TextTokenParameters.getList()" because "parameters" is null
    at dev.pp.parameters.textTokenParameter.TextTokenParameters.merge(TextTokenParameters.java:112)
    at dev.pmlc.ext.utilities.pmltohtml.options.PMLToHTMLOptionsHelper.createMergedOptions(PMLToHTMLOptionsHelper.java:71)
    at dev.pmlc.commands.pmltohtml.PMLToHTMLCommand.convertDocumentNode(PMLToHTMLCommand.java:147)
    at dev.pmlc.commands.pmltohtml.PMLToHTMLCommand$Executor.execute(PMLToHTMLCommand.java:120)
    at dev.pmlc.commands.pmltohtml.PMLToHTMLCommand$Executor.execute(PMLToHTMLCommand.java:39)
    at dev.pp.commands.command.CommandExecutor.execute(CommandExecutor.java:17)
    at dev.pp.commands.command.FormalCommand.execute(FormalCommand.java:159)
    at dev.pp.commands.picocli.PicocliCommandLineExecutor.executeSubCommand(PicocliCommandLineExecutor.java:72)
    at dev.pp.commands.picocli.PicocliCommandLineExecutor.executeCommand(PicocliCommandLineExecutor.java:41)
    at dev.pmlc.commands.PMLCommands.runCommand(PMLCommands.java:29)
    at dev.pmlc.commands.Start.main(Start.java:20)

Please consider sending the above message to the developers of the application, so that this problem can be fixed.
[Finished in 1.7s]
```
Empty options should be acceptable, e.g. for procedurally generated documents which might or might not inject options into the node, and in any case PMLC should at the most produce a warning but still convert the rest of the document (if valid).

**Comment by pml-lang** [comment]
> Empty options should be acceptable, e.g. for procedurally generated documents which might or might not inject options into the node, and in any case PMLC should at the most produce a warning but still convert the rest of the document (if valid).

Yes, PMLC must not crash in case of an empty `options` node. Instead, a warning should be emitted, and execution should continue.

**Comment by pml-lang** [comment]
Fixed in version 4.0.0.
